CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The changes are relative to the previous release, unless the baseline is specifi
 ## [Unreleased]
 
 ### Added since 1.0.0
+* Remove backwardDirection from `avifGainMapMetadata`.
```

Comment:
Please remove this item. The new experimental gain map feature is covered by the next item (line 13). Since this new feature hasn't appeared in a libavif release, we don't need to document changes to this new feature.

Reply:
Removed.